src/main/java/com/spotify/sparkey/IndexHash.java

```diff
@@ -813,4 +810,67 @@ public boolean markSupported() {
       return false;
     }
   }
+
+  /**
+   * Returns dividend % divisor, where the dividend and divisor are treated as unsigned 64-bit
+   * quantities.
+   *
+   * Inlined from Google Guava 20.0
+   *
+   * @param dividend the dividend (numerator)
+   * @param divisor the divisor (denominator)
+   * @throws ArithmeticException if divisor is 0
+   * @since 11.0
+   */
+  private static long remainder(long dividend, long divisor) {
```

Comment:
Is this equivalent of https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Long.html#remainderUnsigned(long,long) ?

Reply:
Will try that!